docs/content/preview/releases/techadvisories/ta-23476.md

```diff
@@ -17,161 +17,105 @@ type: docs
 
 |          Product           |  Affected Versions  |  Related Issues   | Fixed In |
 | :------------------------- | :------------------ | :---------------- | :------- |
-| {{<product "ycql">}}       | {{<release "All">}} | {{<issue 11052>}}, {{<issue 23476>}} | TBD |
+| {{<product "ycql">}}       | {{<release "All">}} | {{<issue 11052>}}, {{<issue 23476>}} | {{<release "2.25.0.0, 2024.1.4.0, 2024.2.1.0, 2.20.9.0">}} |
 
 ## Description
 
-In YCQL, timestamp data inserted with the `currenttimestamp()` function may not be returned for queries with an equality condition on the timestamp. If the timestamp is part of the primary key, this also affects index scans as the table lookup for an index scan internally uses an equality condition to look up the corresponding table rows. The data can still be retrieved as expected by primary key or index-only scan queries with no condition or inequality conditions on the timestamp column.
+In YCQL, timestamp data inserted with the `currenttimestamp()` function or explicit timestamp strings with microseconds precision may not be returned for queries with an equality condition on the timestamp. This can cause discrepancies during queries or index scans on tables where timestamps were part of the primary key. The data can still be retrieved as expected by primary key or index-only scan queries with no condition or inequality conditions on the timestamp column.
 
-The root cause is that in YCQL, the timestamp type has millisecond precision, to align with vanilla Cassandra, ensuring compatibility with Cassandra drivers and tools. However, YCQL can internally store data with microsecond precision, adjusting the precision to milliseconds when data is sent or received through the YCQL API. The discrepancy between the wire protocol's millisecond precision and the storage system's microsecond precision can lead to unexpected behavior.
+The root cause is that in YCQL, the timestamp type has millisecond precision, to align with vanilla Cassandra, ensuring compatibility with Cassandra drivers, and tools. However, YCQL can internally store data with microsecond precision, adjusting the precision to milliseconds when data is sent, or received through the YCQL API. The discrepancy between the wire protocol's millisecond precision and the storage system's microsecond precision can lead to unexpected behavior.
 
 ## Mitigation
 
-There are two key aspects to mitigating the issues caused by the discrepancy between the YCQL wire protocol's millisecond precision and the storage system's microsecond precision:
+### For new writes
 
-1. Ensure new writes use the correct precision.
+For versions before v2.25.0.0, v2024.2.1.0, {{<release "2024.1.4.0, 2.20.9.0">}}, you can adjust the new writes to use the correct precision as follows by modifying your application to use `totimestamp(now())` instead of `currenttimestamp()` to generate millisecond-precision timestamp values that are compatible with the wire protocol.
```

Comment:
```suggestion
For versions before v2.25.0.0, v2024.2.1.0, {{<release "2024.1.4.0">}}, and {{<release "2.20.9.0">}}, you can adjust the new writes to use the correct precision by modifying your application to use `totimestamp(now())` instead of `currenttimestamp()` to generate millisecond-precision timestamp values that are compatible with the wire protocol.
```